### ISSUE TITLE
Empty blocks should keep brackets on the same line

### ISSUE THREAD

**Issue by aheber** [dangmai/prettier-plugin-apex#205]
Using Prettier on Javascript, empty block statements are closed on the same line. In Apex empty block statements are being closed on the next line.

This isn't common except for when you're extending classes, particularly Exceptions which are commonly extended without any additional body.
```apex
public class TestException extends Exception {}
```

```sh
# Prettier options (if any): NONE
```

**Input:**

```apex
public class Test123 {
  public class Test234 {

  }

  public void test789() {

  }
}
```

**Actual output:**

```apex
public class Test123 {
  public class Test234 {
  }

  public void test789() {
  }
}

```

**Expected output:**

```apex
public class Test123 {
  public class Test234 {}

  public void test789() {}
}
```

**Additional information (please fill this out)**:

- OS: Windows
- Version: 10
- Plugin Version:
```
npm ls prettier-plugin-apex
-- prettier-plugin-apex@1.3.0
```

**Comment by aheber** [comment]
Happy to write the PR for this if the change makes sense.

**Comment by dangmai** [comment]
Hi, thank you for the bug report. So this is actually a conscious decision, all the way before 1.0.0 - see this ticket: https://github.com/dangmai/prettier-plugin-apex/issues/15

I don't foresee reverting that decision. I do understand that it is not the best in certain cases (like you point out), but it can't be anyway. An example case where not adding new lines is weird is this: A user has auto format turned on, then create a class like this:

```
class ABC {
}
```

then decide to save before actually adding in the content - then their class would suddenly lose the new line and it would be annoying to have to go and add it again, and remember not to save.

It is also not trivial to handle comments in an empty block, this is actually the easiest way to make sure dangling comments are handled correctly 😄 

Sorry for closing your tickets, I do appreciate your efforts in opening them, please open more 👍 There have been a lot of decisions made for this project a long time ago, and it's good to revisit them once in a while. I'm not opposed to change my mind, especially if a lot of people run into the same pain point.

**Comment by aheber** [comment]
Thanks for pointing out the old issue. I thought I searched but missed that one for some reason (I don't think I was searching for `blocks` but using another term before I remember what they were called).

I'll file this with I understand it but don't prefer it. I will admit that the described problem often happens to me with Javascript and it does mean I have had to change my behaviors but the end result is much better closing on the same line.

Could there be a middle ground, though complicated to code and explain, to say methods always close on a new line but empty classes close on the same line?
